### PR TITLE
[pipeline](CTE) Support multi stream data sink in pipeline

### PR DESCRIPTION
**Pull request by HappenLee**
# Proposed changes

Support CET reuse in BE code

## Problem summary

Describe your changes.

## Checklist(Required)

* [ ] Does it affect the original behavior
* [ ] Has unit tests been added
* [ ] Has document been added or modified
* [ ] Does it need to update dependencies
* [ ] Is this PR support rollback (If NO, please explain WHY)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by HappenLee** [comment]
run buildall

File: be/src/vec/CMakeLists.txt [hunk 1]
```diff
@@ -286,7 +286,7 @@ set(VEC_FILES
   olap/olap_data_convertor.cpp
   olap/vertical_merge_iterator.cpp
   olap/vertical_block_reader.cpp
-  sink/vmysql_result_writer.cpp
+        sink/vmysql_result_writer.cpp
```
Comment (Gabriel39) [inline be/src/vec/CMakeLists.txt:289]:
```suggestion
    sink/vmysql_result_writer.cpp
```

File: be/src/pipeline/pipeline_fragment_context.h [hunk 2]
```diff
@@ -169,6 +169,7 @@ class PipelineFragmentContext : public std::enable_shared_from_this<PipelineFrag
 
     ExecNode* _root_plan = nullptr; // lives in _runtime_state->obj_pool()
     std::unique_ptr<DataSink> _sink;
+    std::vector<std::unique_ptr<DataSink>> _multi_cast_stream_sink_senders;
```
Comment (Gabriel39) [inline be/src/pipeline/pipeline_fragment_context.h:172]: Unify `_multi_cast_stream_sink_senders` and `_sink` ?

**Comment by HappenLee** [comment]
run buildall

**Review by Gabriel39 — APPROVED** [review]
LGTM

**Review by Gabriel39 — APPROVED** [review]
LGTM

**Pull request merged**